templates/monitoring/alertmanager/alertmanager-email-config.tmpl

```diff
@@ -2,35 +2,7 @@
 {{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts?receiver={{ .Receiver | urlquery }}{{ end }}
 
 {{/* custom subject template */}}
-{{ define "__intlysubject" }}
-	
-	{{/* declaring variables */}}
-	{{ $lenOfAlerts := ( len .Alerts ) }} {{ $clustername := "${CLUSTER_NAME}"}} {{ $alertname := "" }} {{ $severity := "" }} {{ $ifCritical := "" }}
-
-  {{/* checking if there is single or multiple alerts */}}
-	{{if eq $lenOfAlerts 1 }} 
-		{{ $clustername }}|{{ .CommonLabels.alertname  }}|({{ .CommonLabels.severity | title }})|{{ .Status | title }}{{ if eq .Status "firing" }}{{ end }}
-	{{ else }}
-		{{if gt $lenOfAlerts 1 }} 
-			{{ $alertname = "Multiple Alerts*" }}
-			{{ if eq .CommonLabels.severity "critical" }} 
-				{{ $ifCritical = "true" }}
-			{{ else }}
-				{{ $severity = ( .CommonLabels.severity ) }} 
-			{{ end }}
-		{{ end }}
-	{{ end }}
-
-	{{/* in case of multiple alerts, checking if the severity of any alert(s) is critical*/}}
-	{{ if eq $alertname "Multiple Alerts*" }}
-		{{ if eq $ifCritical "true" }} 
-			{{ $clustername }}|{{ $alertname }}|(Critical)|{{ .Status | title }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ end }}
-		{{ else }}
-			{{ $clustername }}|{{ $alertname }}|({{ $severity | title }})|{{ .Status | title }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ end }}
-		{{ end }}
-	{{ end }}
-
-{{ end }}
+{{ define "__intlysubject" }}{{ $lenOfAlerts := ( len .Alerts ) }}{{ $clustername := "${CLUSTER_NAME}"}}{{ $alertname := "" }}{{ $severity := "" }}{{ $ifCritical := "" }}{{/* checking if there is single or multiple alerts */}}{{if eq $lenOfAlerts 1 }}{{ $clustername }}|{{ .CommonLabels.alertname  }}|({{ .CommonLabels.severity | title }})|{{ .Status | title }}{{ if eq .Status "firing" }}{{ end }}{{ else }}{{if gt $lenOfAlerts 1 }}{{ $alertname = "Multiple Alerts*" }}{{ if eq .CommonLabels.severity "critical" }}{{ $ifCritical = "true" }}{{ else }}{{ $severity = ( .CommonLabels.severity ) }}{{ end }}{{ end }}{{ end }}{{/* in case of multiple alerts, checking if the severity of any alert(s) is critical*/}}{{ if eq $alertname "Multiple Alerts*" }}{{ if eq $ifCritical "true" }}{{ $clustername }}|{{ $alertname }}|(Critical)|{{ .Status | title }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ end }}{{ else }}{{ $clustername }}|{{ $alertname }}|({{ $severity | title }})|{{ .Status | title }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ end }}{{ end }}{{ end }}{{ end }}
```

Comment:
I think what we should be doing here is using the `{{-` and `-}}` of this templating language, which instructs the templating engine to ignore following or preceding whitespaces. This will let us keep the template readable (for maintenance) and keep the email readable (for actual use). See an example here: https://gist.github.com/geekodour/fe6539d85de65e0da7e3e16543521483

Reply:
The templating language is apparently based heavily on the go templating language, which goes into more detail on the subject here: https://pkg.go.dev/text/template

Reply:
@philbrookes, just re-iterating.

Please review. Thank you!